apps/payments/next/middleware.ts

```diff
@@ -4,6 +4,17 @@
 import { NextRequest, NextResponse } from 'next/server';
 
 export function middleware(request: NextRequest) {
+  // Read env vars directly from process.env
+  // Currently not possible to use app/config in middleware
+  const accountsStaticCdn = process.env.CSP__ACCOUNTS_STATIC_CDN;
```

Comment:
Nit: Could you add a link or a 'As of [date]' so when we visit in future we'll have a better idea of the status and whether it will become possible.